rest/data/admin.go

```diff
@@ -166,9 +167,36 @@ func (ac *DBAdminConnector) RestartFailedTasks(queue amboy.Queue, opts model.Res
 		}
 	}
 
-	return &restModel.RestartTasksResponse{
-		TasksRestarted: results.TasksRestarted,
-		TasksErrored:   results.TasksErrored,
+	return &restModel.RestartResponse{
+		ItemsRestarted: results.ItemsRestarted,
```

Comment:
I don't understand how this gets populated when it's not a dry run
(though I don't think your changes are related)

Reply:
Good question. I believe it doesn't get populated when it's not a dry run, perhaps the information isn't necessary.